modelcard.md

```diff
@@ -18,20 +18,26 @@ metrics:
 model-index:
 - name: {model_id}
   results:
-  - task: 
-      type: {task_type}  # Required. Example: automatic-speech-recognition
-      name: {task_name}  # Optional. Example: Speech Recognition
+  - task:
+      type: {task_type}             # Required. Example: automatic-speech-recognition
+      name: {task_name}             # Optional. Example: Speech Recognition
     dataset:
-      type: {dataset_type}  # Required. Example: common_voice. Use dataset id from https://hf.co/datasets
-      name: {dataset_name}  # Required. Example: Common Voice zh-CN
-      args: {arg_0}         # Optional. Example: zh-CN
+      type: {dataset_type}          # Required. Example: common_voice. Use dataset id from https://hf.co/datasets
+      name: {dataset_name}          # Required. Example: Common Voice (French)
+      config: {dataset_config}      # Optional. Example: fr
+      split: {dataset_split}        # Optional. Example: test
+      revision: {dataset_revision}  # Optional. Example: 5503434ddd753f426f4b38109466949a1217c2bb
+      args: 
+        {arg_0: value_0}            # Optional. Example for wikipedia: language: en
```

Comment:
Not sure if I'm using the brackets {} correctly here, but the aim is to indicate that `args` is a dict of key-value pairs. Following @lhoestq's example, I've used `wikipedia` here